Terminal.Gui/Views/Scroll/Scroll.cs

```diff
@@ -35,6 +30,37 @@ public Scroll ()
 
     private int _size;
 
+    /// <inheritdoc/>
+    public override void EndInit ()
```

Comment:
So, just thinking here...

Should we really have EndInit on the base class `public`?

Seems to me that should be `protected` or, if we have coupling internally outside of its hierarchy, `protected internal` (accessible to everything internal, as well as to derived types both internally and externally).

Reply:
Continuing the thought...

With the Being/End pattern, we are also already set up for making them `async`, as well, which makes even more sense if it remains public.

Reply:
I overridden the `EndInit` instead of using the `Initialized` event because this is a View derived class and where the event is being raised. Any changes on `BeginInit/EndInit` isn't the propose of this PR and you'll need to open an issue for that.

Reply:
Yeah it wasn't meant for this PR. Just thinking out loud for future possibilities.